integration-test/helpers/harness.js

```diff
@@ -171,6 +171,20 @@ export async function defaultIOSScript (page) {
         .applyTo(page)
 }
 
+/**
+ * @param {import("@playwright/test").Page} page
+ */
+export async function defaultAndroidScript (page) {
+    return createAutofillScript()
+        .replaceAll({
+            userPreferences: {
+                platform: {name: 'android'}
+            }
+        })
+        .platform('android')
+        .applyTo(page)
+}
```

Comment:
This is now the same for all tests, so I've centralised it here. Config changes are passed via `withRuntimeConfigOverrides`.